benchmark/camelyon/pure_substrafl/register_assets.py

```diff
@@ -274,13 +221,13 @@ def get_test_data_nodes(
     Returns:
         TestDataNode: Substrafl test data.
     """
-    # only one metric is needed as permissions are public
-    metric_key = asset_keys.get("metric_key") or register_metric(clients[0])
-    asset_keys.update(
-        {
-            "metric_key": metric_key,
-        }
-    )
+
+    def auc(datasamples, predictions_path):
```

Comment:
I'm wondering if we would like to have multiple metrics on this test, since it's the more complex case. 
Then, if we add the documentation examples in the CI, MNIST would cover that case so it's good. What do you think?

Reply:
That a good idea, especially because this is the benchmark we run on the CI in remote mode every night :) I'll add an accuracy metric !